### DISCUSSION TITLE
Clean images in a controller when this controller is destroyed ?

### DISCUSSION THREAD

**Discussion by yellowTiger** [onevcat/Kingfisher#1783] [Q&A]
when a controller call the deinit func , I want to remove images in this controller from memory immediately,how can I do?

**Comment by onevcat** [comment]
Kingfisher does not hold a tracking for which images and its view controller in use. So it is not possible.

If you want a minimal memory usage, maybe just set the [memory cache config](https://github.com/onevcat/Kingfisher/wiki/Cheat-Sheet#set-limit-for-cache) limitation to a very small value to disable it. So Kingfisher won't provide a memory cache.
